src/device/device_declare.hpp

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+
+//#define DEBUGLEDGER
+//#define IODUMMYCRYPT 1
+//#define IONOCRYPT 1
+
+namespace hw {
+    class Device;
+
+    Device& get_device(std::string device_descriptor);
```

Comment:
I need this "pre-declaration" because of init in account.h. 

Maybe be we can avoid this by declaring explicitly the default constructor of account_keys in account.cpp.